### DISCUSSION TITLE
eme_cron_send_queued

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#633] [General]
Hi Franky

my website tool says: Das geplante Ereignis, eme_cron_send_queued, wird zu spät ausgeführt. Deine Website funktioniert noch, aber das kann darauf hindeuten, dass die Planung von Beiträgen oder automatisierten Updates nicht wie beabsichtigt funktioniert.

In English: The scheduled event, eme_cron_send_queued, is executed too late. Your website is still working, but this may indicate that the scheduling of posts or automated updates is not working as intended.

 Please can you check this.

Best Regards

**Comment by liedekef** [comment]
This message means something is wrong with your WP cron. Most likely you're using EME to schedule mails to be sent and also using the normal WP cron. The normal WP cron is not scheduled on fixed times but runs queued events when someone visits the site. EME gives you a hint on how to improve that (just a hint!!):
```
Cron tip for more accurate scheduled actions:
Put something like this in the crontab of your server:*/5 * * * * wget -q -O - http://localhost/wordpress/wp-cron.php >/dev/null 2>&1
And add the following to your wp-config.php: define('DISABLE_WP_CRON', true);
```
If this doesn't help you, do a internet search for DISABLE_WP_CRON and go from there.
